### ISSUE TITLE
Error evaluating expression: Class constructor qe cannot be invoked without 'new'

### ISSUE THREAD

**Issue by tjdaley** [dbgate/dbgate#802]
I am connected to a MongoDB instance. I edited some documents in the 'vendors' collection and then clicked 'update'.

Here is the script that was generated:

```javascript
db.vendors.updateOne({
  "_id": ObjectId("6637f3b78bbcb3312e58dd9d")
}, {
  "$set": {
    "normalized_vendor": "Citi Card"
  }
});
db.vendors.updateOne({
  "_id": ObjectId("6637f3b78bbcb3312e58dd9e")
}, {
  "$set": {
    "normalized_vendor": "M&T Bank",
    "normalized_category": "Mortgage",
    "category": "Mortgage"
  }
});
db.vendors.updateOne({
  "_id": ObjectId("6637f3b78bbcb3312e58dd9f")
}, {
  "$set": {
    "normalized_vendor": "Mohela"
  }
});
db.vendors.updateOne({
  "_id": ObjectId("662a70267b8f325674447158")
}, {
  "$set": {
    "normalized_vendor": "Marriott"
  }
});
db.vendors.updateOne({
  "_id": ObjectId("662e94138cc2f1bf9ccbeae8")
}, {
  "$set": {
    "normalized_vendor": "Zelle"
  }
});
```

When I run the script, I get the error "Error evaluating expression: Class constructor qe cannot be invoked without 'new'"

I highlighted each "updateOne" query and pressed CTRL+SHIFT+ENTER to run it and each one resulted in the same error.

Here is the information from the ABOUT box:

Version: 5.2.8
Build date: 2024-05-16
Web: dbgate.org
Source codes: github
Docker container: docker hub
Online demo: demo.dbgate.org
Search plugins: npmjs.com

**Comment by tjdaley** [comment]
UPDATE: I copied the generated script verbatim into Studio-3T and it ran perfectly. For that reason, I would conclude that the error does not arise from the syntax of the generated script but rather in the execution of the script. Hope that helps.

**Comment by janproch** [comment]
This is already fixed in newest BETA.

Duplicate of this: https://github.com/dbgate/dbgate/issues/798

**Comment by janproch** [comment]
Fix released in 5.2.9
